content/2017/02/06/server-side-rendering-client-side-app.rst

```diff
@@ -0,0 +1,257 @@
+        this.model = new BB.Model(modelData);
+        // Trigger any onRender handlers attached to the view.
+        this.triggerMethod('render', this);
+      }
+    });
+
+    const myView = new ViewWithModel({ el: $('.comment') });
+
+Or for a view with a collection of models:
+
+.. code:: js
+
+    import BB from 'backbone';
+    import Mnt from 'backbone.marionette';
+
+    const MyChildView = Mnt.View.extend({ ... });
+
+    const ViewWithCollection = Mnt.CollectionView.extend({
+      collection: new BB.Collection(),
+      childView: MyChildView,
+      initialize () {
+        if (this.options.el) {
+          this.attachChildren();
+        }
+      },
+      // Look through existing child [data-js-model] elements, adding models
+      // to the collection, and attaching views to the models.
+      attachChildren () {
+        const view = this;
+        const collection = view.collection;
+        const children = this.$('[data-js-model]');
+        children.each((idx, el) => {
+          const $el = $(el);
+          const modelData = $el.data('js-model');
+          let model = collection.get(modelData.id);
+          if (!model) {
+            model = collection.add(modelData, { silent: true });
+          }
+          const childView = new view.childView({ model, el });
+          view.addChildView(childView, idx);
+        });
+        // Set _isRendered to prevent the collectionView from rendering.
+        view._isRendered = true;
+        // Trigger any onRender handlers attached to the view.
+        view.triggerMethod('render', view);
+      }
+    });
+
+    const myView = new ViewWithCollection({ el: $('.comment-list') });
+
+Now we have a model or collection of models instantiated with data from our
+server-rendered markup, being managed by Marionette views! ‎🎉
+
+
+Where Do We Go From Here?
+-------------------------
+
+In the end, we're slowly moving toward the best of both worlds: a
+server-rendered page (easily indexable by search engines, with all the content
+immediately visible to users), with all the client-side benefits of a
+single-page app (live-updating components, no page refreshes).
+
+There are a number of improvements we could make (prioritizing the most
+important pieces of interactivity and lazy-loading the rest, abstracting our
+code into a Marionette `behavior`_ that can be added to any view where we might
+want to pre-load with data from the DOM, etc.), but this gets us most of the
+way there. Every step of the way, we strive to minimize the amount of
+duplicated code or logic — no need for a JavaScript process on the server, and
+no duplicated templates.
+
+.. _behavior: http://marionettejs.com/docs/master/marionette.behavior.html
+
+We have a number of other tricks for sharing canonical data — global settings,
+third-party API keys, minified asset mappings, and even color maps generated
+from Scss — but those will wait for a later installment in this series.
```

Comment:
Based on checking sass-lang.com, I think you're right

![Sass SCSS](https://dl.dropbox.com/s/g63rr9z8v12ygc9/Screenshot%202017-02-03%2015.57.29.jpg)